### PR TITLE
Fixed update-password issues and other minor issues

### PR DESCRIPTION
**Pull request by Usenmfon**

### CODE REVIEW

File: backend/src/database/repositories/ResetTokenRepo.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default class ResetTokenRepo {
     };
 
     static getToken = async (token, userId) => {
-        return connection("tokens").where("userId", userId).first();
+        return connection("tokens").where("userId", userId).andWhere("token", token).first();
```
Comment (Mofe620) [inline backend/src/database/repositories/ResetTokenRepo.js:9]: Please undo this change. The andWhere clause doesn't fix the problem. It compounds the issue a bit

**Review by Mofe620 — CHANGES_REQUESTED** [review]

**Review by Idiono-mfon — APPROVED** [review]
updates reviewed. Great job

File: backend/src/database/repositories/ResetTokenRepo.js (hunk 1 above)
Comment (Usenmfon) [reply backend/src/database/repositories/ResetTokenRepo.js]: Issues have been resolved. 

Source code is up to date.

**Review by orlando70 — APPROVED** [review]

**Review by orlando70 — APPROVED** [review]

**Review by Mofe620 — APPROVED** [review]
Well done Chief. Nice one.
